product_docs/docs/postgres_distributed_for_kubernetes/1/architecture.mdx

```diff
@@ -192,3 +192,9 @@ reliably communicate with each other.
 can be realized on multiple Kubernetes clusters that meet the connectivity
 requirements.
 More information can be found in the ["Connectivity"](connectivity.md) section.
+
+!!! NoteAvailability zones 
+    When creating Kubernetes clusters in different availability zones for cross-regional replication, 
+    ensure the clusters can communicate with each other by enabling network connectivity. To ensure 
+    pods and services are reachable by all Kubernetes clusters, deploy a network connectivity application like 
+    [Submariner](https://submariner.io/) on every cluster. 
```

Comment:
```suggestion

!!! NoteAvailability zones 
    When creating Kubernetes clusters in different availability zones for cross-regional replication, 
    ensure the clusters can communicate with each other by enabling network connectivity. Specifically, every service created with a `-node` or `-group` suffix must be discoverable by all other `-node` and `-group` services. This can be achieved by deploying a network connectivity application like 
    [Submariner](https://submariner.io/) on every cluster. 
```